resolwe_bio/processes/reads_processing/alignmentsieve.py

```diff
@@ -0,0 +1,148 @@
+"""Filter BAM files using alignmentSieve from deeptools."""
+import os
+from pathlib import Path
+
+from plumbum import TEE
+
+from resolwe.process import (
+    Cmd,
+    DataField,
+    FileField,
+    IntegerField,
+    Persistence,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class AlignmentSieve(Process):
+    """Filter alignments of BAM files according to specified parameters.
+
+    Program is bundled with deeptools. See [documentation](
+    https://deeptools.readthedocs.io/en/develop/content/tools/alignmentSieve.html)
+    for more details.
+    """
+
+    slug = "alignmentsieve"
+    name = "alignmentSieve"
+    process_type = "data:alignment:bam:"
+    version = "1.0.0"
+    category = "Read Processing"
```

Comment:
Bam processing or Alignment filtering?

Reply:
Maybe `BAM processing` given that this is an already existing category?